tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -60,11 +60,15 @@ select * from t1 inner join t on t.a = t1.b;
 select * from t2 inner join t on t.a = t2.c;
 select * from t2 inner join t on t.a = t2.c + 1;
 select * from t2 inner join t on t.a = t2.c + 1 and t.a - 1 = t2.c;
+select count(*) from numbers(1000) as t inner join numbers(1000) as t1 on t.number = t1.number;
 
+-- order by
+SELECT number%3 as c1, number%2 as c2 FROM numbers_mt (10) order by c1 desc, c2 asc;
+SELECT number, null from numbers(3) order by number desc;
+SELECT number%3 as c1, number%2 as c2 FROM numbers_mt (10) order by c1, number desc;
```

Comment:
Maybe need test `SELECT COUNT() FROM numbers_mt(10) GROUP BY number ORDER BY SUM(number)`;

Reply:
`SELECT SUM(number) AS s FROM numbers_mt(10) GROUP BY number ORDER BY s`;

Reply:
> Maybe need test `SELECT COUNT() FROM numbers_mt(10) GROUP BY number ORDER BY SUM(number)`;

Nice, the sort expression(s) can be any expression that would be valid in the query's select list, so I need to process its input schema and produce its output schema. Thank you!

Reply:
I haven't figured out how to support aggregate functions in order by, but now that other expressions than aggregate functions are supported, I'll note an issue.